<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetToFileDialog</class>
 <widget class="QWidget" name="NetToFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="urlEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="pathEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>请输入保存的地址：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="checkUrlBtn">
         <property name="text">
          <string>检查链接</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="urlLab">
         <property name="text">
          <string>请输入捕获url地址：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="saveFileBtn">
         <property name="text">
          <string>文件夹选择</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>用户Cookie:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="CookieEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>分辨率：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cb_video_format"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>视频解码格式：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_quality"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>音频解码格式：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="cb_audio_format"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_download">
       <property name="text">
        <string>开始抓取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QListWidget" name="listWidget">
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="layoutMode">
      <enum>QListView::LayoutMode::SinglePass</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
